dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -33,7 +33,8 @@
   %ul
     %li
       It helps us understand who the population of teachers involved in our
-      Computer Science Principles (CSP) PD program are.
+      = @workshop.course
```

Comment:
@tanyaparker is this ok to use the course name, e.g. "CS Principles", or do we want to preserve the spelled out "Computer Science Principles (CSP)" style?